doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -1835,6 +1860,14 @@ NCSDK-27983: No Bluetooth advertising after a software reset
 
   **Workaround:** Press the **RESET** button on the nRF54H20 DK after performing a factory or software reset of the device.
 
+.. rst-class:: v2-8-0
+
+NCSDK-30261: Bluetooth LE connection parameter results in a loop in the :ref:`nrf_desktop_ble_conn_params`
```

Comment:
`Bluetooth LE connection parameter update is triggered multiple times in the :ref:`nrf_desktop_ble_conn_params``